images/admin-builder/Dockerfile

```diff
@@ -45,5 +45,11 @@ RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
 ENV PATH=/workspace/google-cloud-sdk/bin:${PATH}
 RUN gcloud --version
 
+# Install Terraform CLI & check it was installed
+ARG TF_VERSION="0.12.2"
```

Comment:
Not entirely sure what the Terraform binary is supposed to be used for, but if it's for validating/formatting of our stuff will `0.12.x` work? IIRC there are some providers (Vault) that don't work on `0.12.x`?

Reply:
You raise a very good point! It is exactly for `fmt` and `validate` so let me verify it doesn't work and I'll get back to you. I vaguely remember this came up a couple of weeks ago right?

Reply:
0.12 is causing pain, going to 0.11.14